WcaOnRails/app/assets/javascripts/users.js

```diff
@@ -43,4 +43,14 @@ $(function() {
     $unconfirmed_wca_id_profile_link.attr('href', "/results/p.php?i=" + unconfirmed_wca_id);
   });
   $unconfirmed_wca_id.trigger('input');
+
+  // Change bootstrap-table pagination description
+  var $table = $('.bootstrap-table');
+  var options = $table.bootstrapTable('getOptions');
+  options.formatRecordsPerPage = function(pageNumber) {
+    return pageNumber + ' users per page';
+  };
+  options.formatShowingRows = function(pageFrom, pageTo, totalRows) {
+    return 'Showing ' + pageFrom + ' to ' + pageTo + ' of ' + totalRows + ' users ';
+  };
```

Comment:
Note: we don't have to re-render the table with 'refreshOptions', because firstly our table is rendered empty, then our script changes the options and at the end bootstrap-table sends request for json, then renders the table again (with our options set up).